base/install_area_detector.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+
+set -ex
+
+git clone --depth 1 --branch ${AREA_DETECTOR_VERSION} \
+    https://github.com/areaDetector/areaDetector
+
+cd areaDetector
+
+git submodule update --init --depth 1 -j ${JOBS} \
+    ADSupport \
+    ADCore
+
+cd configure
+
+module_releases="
+AREA_DETECTOR=${EPICS_MODULES_PATH}/areaDetector
+ADSUPPORT=${EPICS_MODULES_PATH}/areaDetector/ADSupport
+ADCORE=${EPICS_MODULES_PATH}/areaDetector/ADCore
+"
+
+echo "$module_releases" >> ${EPICS_MODULES_PATH}/../RELEASE
+
+echo "
+EPICS_BASE=${EPICS_BASE_PATH}
+
+$module_releases
+
+ASYN=${EPICS_MODULES_PATH}/asyn
+" > RELEASE.local
+
+ln -s RELEASE.local RELEASE_PRODS.local
+ln -s RELEASE.local RELEASE_LIBS.local
+
+echo "
+BUILD_IOCS=NO
```

Comment:
Why do this instead of appending to the existing file?

I can probably be convinced that it's worth it so we have to properly inspect changes between versions instead of doing the least effort, but I'd like to know if that's the goal :)

Reply:
Yeah, that's the point. If anything changes in this configuration file in the `areaDetector` repo, I'd like to have it explicitly and properly handled and well documented in this repository.

Reply:
I'd like for this rationale to be documented in the commit message, then.